app/tut/ga/usaideach.md

```diff
@@ -0,0 +1,339 @@
+---
+title: Roinnt Gníomhartha Úsáideacha
+next: cluiche
+next-text: Cluiche
+prev: gniomh
+prev-text: Ní Briathra a Dhearbhaíonn Ach Gníomh
+tsn-fname: useful
+---
+
+# Mata
+
+Go dtí seo chonaiceamar go leor gníomhartha difriúla (`codladh`{.setanta}, `scríobh`{.setanta}
+srl.). Sula leanaimid ar aghaidh le níos mó [[gnéithe|features]] *Setanta*, ba chóir dúinn fanacht
+ar feadh nóiméad agus féachaint ar cúpla gníomhartha agus oibritheoirí úsáideacha eile.
```

Comment:
ar roinnt gníomhartha agus oibritheoirí (níos fearr ná "cúpla" agus gramadach níos fusa! Is é an t-ainmfhocal uatha san ainmneach a leanann cúpla i gcónaí.)

Reply:
Aistrithe